### PR TITLE
Add Stories From Playbook talk

### PR DESCRIPTION
**Pull request by povilasv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,9 @@ Most recent publications on top.
 * [SaleMove US System Issue - SaleMove - incident report 2017](https://status.salemove.com/incidents/xf6cr710yrzn)
     * involved: AWS, ELB dynamic IPs, DNS `A` record for master, API server
     * impact: production issues with SaleMove US System
+* [Stories Stories from the Playbook - Google - KubeCon Europe 2018](https://youtu.be/N2JUGnwinbQ)
```
Comment (jgcmarins) [inline README.md]: `Stories Stories`
typo?

**Comment by hjacobs** [comment]
Please sort the entry by time (most recent on top). There are already some entries for KubeCon 2018, so the new entry should be at the same position.

**Comment by povilasv** [comment]
@hjacobs done :)

**Comment by povilasv** [comment]
@hjacobs ping :)

**Comment by hjacobs** [comment]
@povilasv sorry for the delay, will watch it today :smile:

**Comment by hjacobs** [comment]
@povilasv thanks!

**Pull request merged**